README.md

```diff
@@ -1,2 +1,35 @@
 # Git Assignment - Siyuan1995
 
+1.What is an issue?
+issue usually refers to a task, bug, or feature request or anything that requires attention within the project.
+people can communicate over the issue and probably assign works.
+
+2.what is a pull request?
+pull request is to propose the changes to github remote repo, and it notify the developer the changes, and then the 
+code reviewer could decide whether to merge the change or not
+
+3.how do I open a pull request
+just go to your forked repo, then find the "pull request" banner  and then click "New pull request", make sure you 
```

Comment:
Forking is not a necessary step, and you can create PR in a repo created by yourself as well (eg. this one)